test/e2e/fixture/zomes/foo/Cargo.lock

```diff
@@ -1,511 +0,0 @@
-# This file is automatically @generated by Cargo.
```

Comment:
@zo-el why delete Cargo.lock?

Reply:
Oh, I get it, because there's a top-level one now

Reply:
That was an oversight.